doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+
+Update kernel image and modules of Pre-launched VM
+**************************************************
+
+Step1: On your development workstation, clone the ACRN kernel source tree, and build the Linux kernel image that will be used to boot the privileged VMs::
+
+  $ git clone https://github.com/projectacrn/acrn-kernel.git
+  Cloning into 'acrn-kernel'...
+  ...
+  $ cd acrn-kernel
+  $ cp kernel_config_uos .config
+  $ make olddefconfig
+  scripts/kconfig/conf  --olddefconfig Kconfig
+  #
+  # configuration written to .config
+  #
+  $ make
+  $ make modules_install INSTALL_MOD_PATH=out/
+
+The last two commands built the bootable kernel image in the arch/x86/boot/bzImage, and loadable kernel modules under the ``./out/`` folder. Copy these files to a removable disk for installing on the KBL NUC i7 board later.
+
+Step2: Current ACRN logical partition scenario implementation requires a multi-boot capable bootloader to boot both ACRN hypervisor and the bootable kernel image built from the previous step. You could install Ubuntu OS to the KBL NUC i7 board on the NVMe SSD by following this Ubuntu tutorial. The Ubuntu installer creates 3 disk partitions on the on-board NVMe SSD. By default, the GRUB bootloader is installed on the ESP (EFI System Partition) partition, which will be used to bootstrap ACRN hypervisor.
+
+Step3: After installing the Ubuntu OS, power off the KBL NUC i7 board, attach the SATA disk and the USB flash disk to the board. Power on the board and make sure it boots the Ubuntu OS from the NVMe SSD, then copy the loadable kernel modules built in Step 1 to the ``/lib/modules/`` folder on both the mounted SATA disk and USB disk. For example, assuming the SATA disk and USB flash disk are assigned to ``/dev/sda`` and ``/dev/sdb`` respectively, the following commands set up the partition mode loadable kernel modules onto the root file systems to be loaded by the pre-launched VMs::
+
+  # Mount the Clear Linux OS root filesystem on the SATA disk
+  $ sudo mount /dev/sda3 /mnt
+  $ sudo cp -r <kernel-modules-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+  $ sudo umount /mnt
+
+  # Mount the Clear Linux OS root filesystem on the USB flash disk
+  $ sudo mount /dev/sdb3 /mnt
+  $ sudo cp -r <path-to-kernel-module-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+  $ sudo umount /mnt
+
+Step 4: Copy the bootable kernel image to the /boot directory::
+
+  $ sudo cp <path-to-kernel-image-built-in-step1>/bzImage /boot/
+
+
+Note: If you update kernel image (bzImage) and modules on ubuntu of KBL NUC i7, need use scp to copy it from the remote host building machine.
+If you install EBtool Docker on the ubuntu of KBL NUC i7, it will be easy to update kernel image and modules.
```

Comment:
Where is EBtool Docker? Can you help to make it clearer?

Reply:
Remove EBtool Docker for doc v2 version, as the the EBtool docker is only internal used.

Reply:
Remove EBtool Docker for doc v2 version, as the the EBtool docker is only internal used.